src/ops/watch.rs

```diff
@@ -19,10 +43,19 @@ pub fn main(project: Project) -> OpResult {
     };
 
     for msg in rx {
-        println!("{:#?}", msg);
+        print_build_message(msg);
+        let _ = std::io::stdout().flush();
     }
 
     build_thread.join().unwrap();
 
     ok()
 }
+
+// TODO: this should probably be something more structured
```

Comment:
Let's drop the TODO. We'll know it needs to be TO-DONE when we get to the feature:

```suggestion
```